restaurant_bill/src/scripts/calculate.js

```diff
@@ -0,0 +1,7 @@
+const calculateBill = (state) => {
+  const { people, totalBill, extraTip } = state;
```

Comment:
`calculateBill` doesn't need to care about the full `state` object. You should only pass in `people`, `totalBill` and `extraTip` as separate parameters in this function.